app/src/main/java/io/github/danthe1st/jdoc4droid/activities/list/javadocs/ListJavadocsActivity.java

```diff
@@ -51,12 +51,37 @@
 
     private ProgressBar progressBar;
 
+    @WorkerThread
```

Comment:
In my opinion, methods shouldn't necessarily ordered in a way that static methods are put before non-static methods but methods should typically be located below their usages.